frontend/simple/views/Join.vue

```diff
@@ -153,11 +158,15 @@ export default {
       }
     }
   },
+  computed: {
+    contract () {
+      return this.$store.state[this.$route.query.groupId]
+    }
+  },
   data () {
     return {
-      contract: { members: [] },
-      inviteHash: null,
-      errorMsg: null
+      errorMsg: null,
+      contract: { members: {} }
```

Comment:
Ah ok. One thing though, `members` is a list, and here's it's defined as an object